modules/check_in/app/controllers/check_in/v2/pre_check_in_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module CheckIn
+  module V2
+    class PreCheckInController < CheckIn::ApplicationController
```

Comment:
Rails controller names are plural by convention: https://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention

Reply:
Interesting to know about the rails convention.  Thanks for the reference link.